contributing-guides/style-guide.md

```diff
@@ -182,15 +182,12 @@ In this case, provide a note and method to determine whether the command current
 
 ## Option syntax
 
-- Use **GNU-style long options** (like `--help` rather than `-h`) when they are cross-platform compatible (intended to work the same across multiple platforms).
+- For commonly/frequently used commands (e.g. `grep`, `tar`, `etc`), we prefer using short options along with [mnemonics](#short-option-mnemonics).
+- For highlighting both long and short options, combine them within a placeholder i.e. `-{{o|-output}}`.
```

Comment:
If we put the first hyphen outside, that kind of implies the long option is `---output` (3 hyphens).
```suggestion
- For highlighting both long and short options, combine them within a placeholder i.e. `{{-o|-output}}`.
```

Reply:
Actually no, because there is one inside and one outside, which makes it 2 hypens

Reply:
Oh, I don't know what I was thinking.

How about this then?
```suggestion
- For highlighting both long and short options, combine them within a placeholder i.e. `{{-o|--output}}`.
```

One hyphen outside looks kind of ambiguous in a client at first glance.
![screenshot](https://github.com/tldr-pages/tldr/assets/126529524/a421fef4-4b81-4b6c-bf7f-309ba71aa29d)